pkg/util/trace/impl/motrace/mo_trace_test.go

```diff
@@ -460,7 +460,12 @@ func TestMOSpan_doProfile(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			_, s := tt.fields.tracer.Start(tt.fields.ctx, "test", tt.fields.opts...)
+			ms, _ := s.(*MOSpan)
+			t.Logf("span.LongTimeThreshold: %v", ms.LongTimeThreshold)
+			time.Sleep(time.Millisecond)
```

Comment:
sleep 1 ms here?

Reply:
Actually, there is no need to sleep.
But i got ut failed result, so i add little code to make sure all well-known conditions are achieved.